Desktop basic java

Bhushan-Thombre opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to project-guidance here: https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

YES

### Description

Notepad is a word processing program which allows changing of text in computer file, Notepad is a simple text editor for basic text editing which enables computer users to create files.

Fixes #478 

### Checklist



- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

photon149 reviewed (approved):
@Bhushan-Thombre Looks good
No Duplicity ✔️ 
@prakhar3062 @Kushal997-das have a look

Kushal997-das reviewed (changes requested):
@Bhushan-Thombre Hello , Please remove wanted files from this PR , And can you please specify you did you added `windows image` . ?

Bhushan-Thombre commented:
I have already removed all the unwanted files and folders from the project. The application contains the about us section where the windows image is used. That's why I added the windows image.
